satellite/overlay/benchmark_test.go

```diff
@@ -302,31 +302,31 @@ func BenchmarkNodeSelection(b *testing.B) {
 
 		b.Run("SelectStorageNodes", func(b *testing.B) {
 			for i := 0; i < b.N; i++ {
-				selected, err := overlaydb.SelectStorageNodes(ctx, SelectCount, criteria)
+				selected, err := overlaydb.SelectStorageNodes(ctx, SelectCount, SelectCount, criteria)
```

Comment:
Shouldn't this be `SelectCount, 0`?

Reply:
okay I can change this and the one below- I thought you wanted a case where both reputable storage nodes and new storage nodes were selected. not only cases where they're separate.

Reply:
These are specifically about measuring them separately. `FindStorageNodes` measures both together.

If you change what these benchmarks returns then they aren't comparable before and after, since they do different amount of work.